crates/ra_ide/src/lib.rs

```diff
@@ -147,7 +147,7 @@ pub struct Assist {
     pub id: AssistId,
     pub label: String,
     pub group_label: Option<String>,
-    pub source_change: SourceChange,
+    pub source_change: Option<SourceChange>,
```

Comment:
I think it's better to split `SourceChange` out of this struct completely. Than, I think we should be able to just `pub use ra_assists::Assist` instead of defining a mirrored struct.